tools/local_development/README.md

```diff
@@ -0,0 +1,20 @@
+[![](https://img.shields.io/badge/unicorn-approved-ff69b4.svg)](https://www.youtube.com/watch?v=9auOCbH5Ns4)
+![][license img]
+
+#### build
+From /tools/local_development directory.
+```bash
+docker build -t smf_base . 
+```
+#### run
+```bash
+docker run -p 7000:7000 -it smf_base bash
+```
+now, you have SMF build and sample apps, you can run for example demo_server, just type
+```bash
+./demo_apps/demo_server -c 1 --ip 0.0.0.0 --port 7000 &
+```
+
+As a client you can use SMF-JAVA https://github.com/KowalczykBartek/smf-java
```

Comment:
Nice! 

Lets also post a command for the client

```
./demo_apps/demo_server -c 1 --ip 0.0.0.0 --port 7000
```

Reply:
added